pkg/reconciler/events/cloudevent/testing.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cloudevent
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+	"time"
+
+	"k8s.io/client-go/tools/record"
+)
+
+// CheckEvents checks that the events received by the FakeRecorder are the same as wantEvents,
+// in the same order.
+func CheckEvents(t *testing.T, fr *record.FakeRecorder, testName string, wantEvents []string) error {
```

Comment:
just realised that `cloudevents.CheckEvents` is not checking cloudevents, which seems like a bit of a misnomer given the package's name. Suggest moving this func to the `pkg/reconciler/events` package if possible.

Reply:
I agree that ideally we'd have CheckEvents in the events package and CheckCloudEvents in the cloudevents package. However it looks like the events package is already doing some [cloudevent related things](https://github.com/tektoncd/pipeline/blob/main/pkg/reconciler/events/event.go#L45-L49) and relies on [checkCloudEvents](https://github.com/tektoncd/pipeline/blob/main/pkg/reconciler/events/event_test.go#L274-L277) in tests, which would cause a circular import if I split up tests in this way. 
It seems like the events package may just need to be refactored but that's probably out of scope for this PR-- I'll create an issue to track.